cpp.mk

```diff
@@ -65,6 +72,12 @@ DEPENDS += $${$1_DEPS}
 CLEANS += $${BUILD_DIR}/bin/$1
 CLEAN_DIRS += $${$1_OBJECT_DIR}
 
+.PHONY: clangtidy_$1 clangtidy_generate_$1
+clangtidy_generate_$1: $${$1_CLANGTIDY}
+
+clangtidy_$1: clangtidy_generate_$1
+	cat $${$1_CLANGTIDY}
+
```

Comment:
I'm happy to add this as an optional thing to run without getting agreement from the team, but I think making it a mandatory part of our builds is much less reasonable and needs a lot more consensus.